README.md

```diff
@@ -61,6 +58,29 @@ Enterprise servers can be configured via the `"deephaven.enterpriseServers"` set
 
 ![Enterprise Server Settings](./docs/assets/dhe-settings.gif)
 
+## SSL Certificates
+Deephaven servers using self-signed certificates or internal CA's will require configuring VS Code to trust the signing certificate.
+
+1. Save the signing certificate in PEM format somewhere on the machine running VS Code. Multiple certificates can be concatenated together in the same file if there are multiple certs that need to be configured.
+1. Set the `NODE_EXTRA_CA_CERTS` environment variable to the path of the signing certificate.
```

Comment:
```suggestion
2. Set the `NODE_EXTRA_CA_CERTS` environment variable to the path of the signing certificate.
```

Reply:
@mofojed Is this change necessary? Markdown rendererer "should" auto number these.

Reply:
You can see the output in the branch:
https://github.com/deephaven/vscode-deephaven/tree/DH-18191-2_logs-and-docs?tab=readme-ov-file#ssl-certificates

Reply:
Yes it does auto number them, was a nitpick.